server/index.js

```diff
@@ -10,9 +10,7 @@ const app = express();
 
 /* MIDDLEWARE */
 app.use(morgan('dev'));
-app.use(express.json({
-  limit: '20mb',
-}));
+app.use(express.json({ limit: '32mb' }));
```

Comment:
is 32mb max size for imgbb?

Reply:
yes ![image](https://user-images.githubusercontent.com/116389520/217328731-c7cc528e-6a4b-42ab-93d1-6c6d2475cf2d.png)